agent/build.gradle

```diff
@@ -55,7 +55,6 @@ clean {
   delete 'plugins_work'
   delete 'agent-plugins.zip'
   delete 'tfs-impl.jar'
-  delete 'logs'
```

Comment:
Why not delete logs? We do it for server.

Reply:
See `cleanLogs` task added to all projects.